src/client/datascience/ipywidgets/commonMessageCoordinator.ts

```diff
@@ -67,7 +67,7 @@ export class CommonMessageCoordinator {
                 payload: any;
             }>();
         this.disposables = this.serviceContainer.get<IDisposableRegistry>(IDisposableRegistry);
-        this.jupyterOutput = this.serviceContainer.get<IOutputChannel>(IOutputChannel, JUPYTER_OUTPUT_CHANNEL);
+        this.jupyterOutput = this.serviceContainer.get<IOutputChannel>(IOutputChannel, STANDARD_OUTPUT_CHANNEL);
```

Comment:
JUPUYTER_OUTPUT_CHANEL === STANDARD_OUTPUT_CHANNEL
This is a carry over from Python exension when we had two, in Jupyter they are the same.